src/lib.rs

```diff
@@ -353,6 +356,7 @@ static EMPTY_CHUNK: EmptyChunkFooter = EmptyChunkFooter(ChunkFooter {
 });
 
 impl EmptyChunkFooter {
+    #[inline]
```

Comment:
This isn't even a public function. Were you seeing that it was not being inlined when profiling or something?